### PR TITLE
Implement `read --follow`

### PR DESCRIPTION
**Pull request by 0x2b3bfa0**
* Requirement for #633

### CODE REVIEW

**Review by dacbd — APPROVED** [review]
Simple and works just fine. Can certainly have a follow-up that is "streaming" logs.

File: cmd/leo/read/read.go [hunk 1]
```diff
@@ -62,39 +68,69 @@ func (o *Options) Run(cmd *cobra.Command, args []string, cloud *common.Cloud) er
 		return err
 	}
 
-	if err := tsk.Read(ctx); err != nil {
-		return err
-	}
+	for last := 0; ; {
```
Comment (tasdomas) [inline cmd/leo/read/read.go:71]:
```suggestion
        var last int
	for {
```

File: cmd/leo/read/read.go [hunk 2]
```diff
@@ -62,39 +68,69 @@ func (o *Options) Run(cmd *cobra.Command, args []string, cloud *common.Cloud) er
...
+			return err
+		}
+
+		status, err := o.getStatus(ctx, tsk)
+		if err != nil {
+			return err
+		}
 
-	return nil
+		if delta := strings.Join(logs[last:], "\n"); delta != "" {
```
Comment (tasdomas) [inline cmd/leo/read/read.go:86]: This is a really complicated way of saying ` if len(logs) > last {`

File: cmd/leo/read/read.go (hunk 1 above)
Comment (tasdomas) [reply cmd/leo/read/read.go:71]: Slightly more idiomatic this way

**Review by tasdomas — APPROVED** [review]

File: cmd/leo/read/read.go (hunk 1 above)
Comment (0x2b3bfa0) [reply cmd/leo/read/read.go]: Thanks! 🙏

File: cmd/leo/read/read.go (hunk 2 above)
Comment (0x2b3bfa0) [reply cmd/leo/read/read.go:87]: This accounts for cases where `logs := []string{""}` and an extra newline would be printed in the beginning. Are you sure it's equivalent? 🤔

Comment (tasdomas) [reply cmd/leo/read/read.go:87]: not sure I understand what you mean - are you trying to work around the case where empty log lines are appended?

Comment (0x2b3bfa0) [reply cmd/leo/read/read.go:87]: Sort of. It appears that the first returned value of `logs` is no t an empty slice, but a slice containing an empty string.

Comment (tasdomas) [reply cmd/leo/read/read.go:87]: I see, let's leave it as is then

**Review by dacbd — APPROVED** [review]

**Pull request merged**